benchmarks/perf-tool/add-filters-to-dataset.py

```diff
@@ -0,0 +1,172 @@
+import getopt
+import os
+import random
+import sys
+
+import h5py
+
+from osb.extensions.data_set import Context, HDF5DataSet
+
+"""
+Script builds complex dataset with additional attributes from exiting dataset that has only vectors. 
```

Comment:
Could you mention what the schema or mapping of the input and output data set might look like?

Reply:
ack